setup.py

```diff
@@ -25,6 +25,7 @@ def _setup_long_description() -> Tuple[str, str]:
         'openai',
         'requests',
         'transformers',
+        'click'
```

Comment:
Can we get click and the other base fixes broken out as a separate diff so we can land that quickly and populate it across the rest of the open diffs?